index.d.ts

```diff
@@ -0,0 +1,131 @@
+import {EventEmitter} from "events";
+
+declare module testableUtils {
+    var isLocal: boolean;
+    var isSmokeTest: boolean;
+    var events: EventEmitter;
+
+    function execute(func: Function)
+
+    function stopwatch(func: Function, param1?: string, param2?: string)
```

Comment:
Rename to `code`, `metricName`, and `resource` as the parameter names.

Reply:
My code messed up, i was working inside node module and when I started agent, it reinstalled my typescript folder modules. Had to do all again, and this got missed